### DISCUSSION TITLE
Brilliant. Simply brilliant!

### DISCUSSION THREAD

**Discussion by CommanderB** [ybizeul/StreamDeckWS#9] [General]
This turbocharges StreamDeck to do ANYTHING. Pretty much anything is possible with Node-Red and providing the hook from StreamDeck to Node-Red (or other websocket client) makes StreamDeck capable of controlling anything. 

I had a kludge of AppleScript calling curl, mosquito (MQTT) and more with about 15 profile pages simulating button icon changes for state changes. All very complicated and unecessary. Now a websocket call to a Node-Red server (that I already had running) and the integration is simple. And much more is possible now.

**Comment by ybizeul** [comment]
You're the perfect user for that project :-)
That's exactly to avoid that kind of mess I created this !

Let me know if you have any ideas for improvements !
